File: AD-HOC/1104/main.dart
import 'dart:io';
import 'dart:math';

void main() {
  var [a, b] = stdin.readLineSync()!.split(' ').map(int.parse).toList();
  while (a != 0 && b != 0) {
    Set<int> a_set = {...stdin.readLineSync()!.split(' ').map(int.parse)};
    Set<int> b_set = {...stdin.readLineSync()!.split(' ').map(int.parse)};
    Set<int> union_set = a_set.union(b_set);
    int difference = min(union_set.length - a_set.length, union_set.length - b_set.length);
    print(difference);
    [a, b] = stdin.readLineSync()!.split(' ').map(int.parse).toList();
  }
}

File: AD-HOC/1104/main.py
a, b = tuple(map(int, input().split()))
while(a != 0 and b != 0):
    a_set = set(map(int, input().split()))
    b_set = set(map(int, input().split()))
    union_set = a_set.union(b_set)
    difference = min(len(union_set) - len(a_set), len(union_set) - len(b_set))
    print(difference)
    a, b = tuple(map(int, input().split()))
